src/main/java/cs2103/v15_1j/jimjim/command/ChangeCommand.java

```diff
@@ -79,50 +90,65 @@ public UIFeedback execute(DataLists searchResultsList, DataLists masterList,
 							  Storage storage, Searcher searcher, Stack<UndoableCommand> undoCommandHistory,
 							  Stack<UndoableCommand> redoCommandHistory) {
 		try {
-			backup = masterList.getTaskEvent(taskNum, prefix);
+			TaskEvent temp = masterList.getTaskEvent(taskNum-1, prefix);
+			if (temp instanceof FloatingTask) {
+				backup = (FloatingTask) temp;
```

Comment:
all these casting are redundant. Doesn't change anything because backup is of type `TaskEvent`